app/src/main/java/ar/com/wolox/android/cookbook/mercadopago/MercadoPagoItemsAdapter.kt

```diff
@@ -0,0 +1,29 @@
+package ar.com.wolox.android.cookbook.mercadopago
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import ar.com.wolox.android.cookbook.R
+import ar.com.wolox.android.cookbook.databinding.ViewMercadopagoItemBinding
+import ar.com.wolox.android.cookbook.mercadopago.model.Product
+
+class MercadoPagoItemsAdapter(
+    private val items: MutableList<Pair<Product, Int>>
+) : RecyclerView.Adapter<MercadoPagoItemsAdapter.ViewHolder>() {
```

Comment:
I'm trying to promote the use of `ListAdapter`. Check if you can extend from that class.